src/lib/mongoBackend/MongoCommonRegister.cpp

```diff
@@ -173,31 +168,25 @@ static bool addTriggeredSubscriptions
   // sub document could include both isPattern=true and isPattern=false documents
   //
   std::string idJsString = "[ ";
-
   for (unsigned int ix = 0; ix < idJsV.size(); ++ix)
   {
+    idJsString += "\"" + idJsV[ix] + "\"";
+
     if (ix != idJsV.size() - 1)
     {
-      idJsString += "\""+idJsV[ix]+ "\" ,";
-    }
-    else
```

Comment:
This goes beyond a style check, but I understand has the same effect and the code is more concise an clearer...

Regression test (functest) will ensure that.

NTC (just informative)